server/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
     compile "org.apache.commons:commons-math3:${versions.commonsmath}"
     compile "com.fasterxml.jackson.dataformat:jackson-dataformat-csv:${versions.jackson}"
     compile "com.fasterxml.jackson.core:jackson-databind:${versions.jackson}"
+    compile "org.postgresql:postgresql:${versions.jdbc}"
```

Comment:
The `PSQLException` type is required.

Reply:
Why is that type used on the server side? I don't think we should depend on the PostgreSQL jdbc in the server module.

Reply:
My mistake, you are right. This came over from the error handling pr, not relevant for this pr.